Examples/MAX78000/CNN/kws20_demo/main.c

```diff
@@ -642,6 +681,44 @@ int main(void)
                 Max = 0;
                 Min = 0;
                 //------------------------------------------------------------
+
+#ifdef SEND_MIC_OUT_SDCARD
+                /**
+                 *
+                 *  - Blink Green led if a keyword is detected
+                 *  - Blink Red led if an unkown keyword is detected
+                 *  - Blink Yellow if detection is low confidence
+                 *  - Solid Red if there is error with SD card interface
+                 *
+                 **/
+				LED_Off(LED_GREEN);
+                if (!ret) {
+                    // Low Confidence
+                    LED_On(LED_GREEN);
+                    LED_On(LED_RED);
+                }
+                if (out_class == 20)   // Unknown class
+                    LED_On(LED_RED);
+
+				int i = 0;
+				for (i=0; i<SAMPLE_SIZE; i++) {
+					// printf("%d\n",serialMicBuff[(serialMicBufIndex+i)%SAMPLE_SIZE]);
+                    snippet[i] = serialMicBuff[(serialMicBufIndex+i)%SAMPLE_SIZE];
+                }
+                snprintf(fileName, sizeof(fileName),"%04d_%s", fileCount, keywords[out_class]);
+                if (writeSoundSnippet((char*)fileName, snippetLength,  &snippet[0]) != E_NO_ERROR) {
```

Comment:
This always creates a file, even for low confidence detections:

![image](https://user-images.githubusercontent.com/38844790/197027075-8d2b99ef-4dd8-47bc-8319-71eaf4e6d322.png)

Reply:
Fixed in https://github.com/Analog-Devices-MSDK/msdk/pull/191/commits/4b1971a0b8aec9b09e15f15a0172ea498917f583